azure-pipelines.vrt-pr.yml

```diff
@@ -28,6 +28,8 @@ jobs:
 
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27057&commitId=2b87ae28e8628badbe452cd383c77c3a65b01074&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1256 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Dropdown Example Teams Dark.Focuses first item.chromium.png|637| Changed|
|Dropdown.Dropdown Example Teams Dark.Focuses second item.chromium.png|637| Changed|
|Dropdown.Dropdown Example Teams Dark.Focuses trigger.chromium.png|596| Changed|
|Dropdown.Dropdown Example Teams Dark.chromium.png|641| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>